velox/exec/fuzzer/WindowFuzzer.h

```diff
@@ -77,20 +77,13 @@ class WindowFuzzer : public AggregationFuzzerBase {
 
  private:
   struct SortingKeyAndOrder {
-    std::string key_;
-    std::string order_;
-    std::string nullsOrder_;
+    const std::string key_;
+    const core::SortOrder sortOrder_;
 
     SortingKeyAndOrder() = delete;
 
-    SortingKeyAndOrder(
-        const std::string& key,
-        const std::string& order,
-        const std::string& nullsOrder) {
-      key_ = key;
-      order_ = order;
-      nullsOrder_ = nullsOrder;
-    }
+    SortingKeyAndOrder(const std::string key, const core::SortOrder sortOrder)
```

Comment:
Hi @pramodsatya, let's remove the `const` keyword. std::move on const variables leads to const && that doesn't match std::string and SortOrder's move constructor.

Reply:
Fixed, thanks for pointing it out.